src/test/java/uk/nhs/digital/nhsconnect/lab/results/fixtures/PathologyRecordFixtures.java

```diff
@@ -6,9 +6,12 @@
 @SuppressWarnings("checkstyle:hideutilityclassconstructor")
 public final class PathologyRecordFixtures {
 
-    public static PathologyRecord generatePathologyRecord(Practitioner requester) {
+    public static PathologyRecord generatePathologyRecord(
+        Practitioner requester,
+        Practitioner performer) {
```

Comment:
When the parameter list is at the same level of indentation as the method body, it's less clear at first glance where the body actually starts. In this case, the parameter list types and names are short enough for one line
```suggestion
    public static PathologyRecord generatePathologyRecord(Practitioner requester, Practitioner performer) {
```

Reply:
Done in b4a72f8